### PR TITLE
VB-290: fix lot section - small description text

### PR DESCRIPTION
**Pull request by Dmytro-Dmytro**

### CODE REVIEW

File: packages/frontend/components/product/card/styles.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export const productDescription = ({
   font-size: ${fontSizes.body2};
   font-weight: ${fontWeights.body2};
   word-break: break-all;
+  min-height: 2.01rem;
```
Comment (voonya) [inline packages/frontend/components/product/card/styles.ts:39]: use sizes from theme

File: packages/frontend/components/product/card/styles.ts [hunk 2]
```diff
@@ -30,12 +30,15 @@ export const productDescription = ({
...
   fontWeights,
+  lineHeights,
 }: Theme) => css`
   margin-bottom: 15px;
   color: ${colors.text};
   font-size: ${fontSizes.body2};
   font-weight: ${fontWeights.body2};
+  line-height: ${lineHeights.body2};
   word-break: break-all;
+  min-height: ${2*/\d+/.exec(lineHeights.body2)}px;
```
Comment (Nightlngale) [inline packages/frontend/components/product/card/styles.ts:41]: I believe `${2*lineHeights.body2}px;` will work

File: packages/frontend/components/product/card/component.tsx [hunk 3]
```diff
@@ -57,7 +57,9 @@ export const ProductCard = (props: ProductCardProps) => {
         </div>
       </div>
       <div>
-        <div css={productName}>{props.name}</div>
+        <div css={productName}>
+		  <StringCutter>{props.name}</StringCutter>
```
Comment (Nightlngale) [inline packages/frontend/components/product/card/component.tsx:61]: Formating

**Review by Nightlngale — CHANGES_REQUESTED** [review]

File: packages/frontend/components/product/card/component.tsx [hunk 4]
```diff
@@ -57,7 +57,9 @@ export const ProductCard = (props: ProductCardProps) => {
         </div>
       </div>
       <div>
-        <div css={productName}>{props.name}</div>
+        <div css={productName}>
+          <StringCutter>{props.name}</StringCutter>
+		</div>
```
Comment (Nightlngale) [inline packages/frontend/components/product/card/component.tsx:62]: Formating

**Review by Nightlngale — APPROVED** [review]

**Review by voonya — APPROVED** [review]

**Pull request merged**